glass/core/array.py

```diff
@@ -166,7 +166,7 @@ def trapezoid_product(
     y = np.interp(x, *f)
     for f_ in ff:
         y *= np.interp(x, *f_)
-    return np.trapezoid(y, x, axis=axis)  # type: ignore[return-value]
+    return np.trapezoid(y, x, axis=axis)
```

Comment:
The failing `ignore[return-value]` checks were on numpy 2.2.0, and should be resolved now.